pocs/base.py

```diff
@@ -1,20 +1,19 @@
 import sys
 
-from . import __version__
-from .utils import config
-from .utils.database import PanMongo
-from .utils.logger import get_root_logger
+from pocs import __version__
+from pocs.utils import config
+from pocs.utils.database import PanMongo
+from pocs.utils.logger import get_root_logger
 
 # Global vars
 _config = None
-_logger = None
 
 
 class PanBase(object):
 
-    """ Base class for other classes within the Pan ecosystem
+    """ Base class for other classes within the PANOPTES ecosystem
 
-    Defines common properties for each class (e.g. logger, config)self.
+    Defines common properties for each class (e.g. logger, config).
```

Comment:
We should soon elaborate on those common properties... Or maybe put them in another object shared by PanBase instances.